wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -1,4 +1,5 @@
 
+require 'wagn/all'
 require 'rails/application'
```

Comment:
so these rake tasks are probably the trickiest part here, because:
1. we only want to represent most of them in one place (somewhere in the decko gem), BUT
2. when called from an app deck we want to require decko/application (currently wagn/application), AND
3. when called from an engine deck we probably just want decko/engine

ALSO, our rake files are just generally messy and confusing.  So it's awesome that this work will inspire a major cleanup!!!

This may or may not be helpful, but there's the added complication that some of these tasks are really only intended for core developers (bootstrap stuff, for example) and really shouldn't show up at all.

I would offer these rough ideas:
1. it's ok with me if we go ahead and start namespacing these as "decko" rather than "wagn".  The word "rake" doesn't even appear on our standard README; all of these things are basically hidden API at this point.
2. one idea is that the require here is determined by an ENV variable.  eg, if ENV['ENGINE_DECK'] is true, then only decko/engine (and not decko/application) is required.  then we could set it up so that any time you do anything like `decko-rails seed`, it effectively calls `env ENGINE_DECK=true rake decko:seed`.
3. it's possible that this handling is done in decko/all, in which case I would understand why we want that file around!